databases/mycli/Portfile

```diff
@@ -1,14 +1,12 @@
-# $Id$
-
 PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        dbcli mycli 1.5.2 v
+github.setup        dbcli mycli 1.8.1 v
 
-python.default_version     27
+python.versions     27 34 35
```

Comment:
I admit that I'm confused myself, but for ports that are not python modules themselves (that is: ports that are not `python/py-foo`) I fail to see how providing multiple `python.versions` would help exactly unless you provide variants to switch to a different python version.

Given the set of files that this port provides and in case that modules are useful for anything else than just `mycli` command line itself, you could also make this a regular python module `py-mycli` and then `py27-mycli`, `py35-mycli` could coexist. Then you could add just a tiny port `mycli` to make a symlink from `$prefix/bin/mycli` to `$prefix/Library/Frameworks/Python.framework/Versions/X.Y/bin/mycli`. That's outside of scope of this PR anyway.

Basically the only question I'm concerned about is what exactly this change solves. (I don't know myself. I don't see how to switch to version 3.5 for example and even if it was possible, it would be kind of violation of conventions.)